doc/source/whatsnew/v1.1.4.rst

```diff
@@ -28,6 +28,7 @@ Fixed regressions
 Bug fixes
 ~~~~~~~~~
 - Bug causing ``groupby(...).sum()`` and similar to not preserve metadata (:issue:`29442`)
+- Bug in :meth:`GroupBy.fillna` that caused a performance regression between 1.0.5 and 1.1.X (:issue:`36757`)
```

Comment:
you don't need the 1.1.X here (as this is the file its in)

Reply:
Done